scripts/datastore_upgrade.py

```diff
@@ -409,6 +411,32 @@ def all_services_started(status_dict):
       return False
   return True
 
+def ensure_cassandra_nodes_match_replication(keyname):
+  # Get the database master IP.
+  with open(MASTERS_FILE_LOC) as masters_file:
```

Comment:
Yes, Thanks there was an existing method in appscale_info for both getting the master ip here and replication factor below!